lib/msal-browser/src/interaction_client/BaseInteractionClient.ts

```diff
@@ -84,155 +84,189 @@ export abstract class BaseInteractionClient {
+    apiId: number,
+    clientId: string,
+    correlationId: string,
+    browserStorage: BrowserCacheManager,
+    logger?: Logger,
+    forceRefresh?: boolean
+): ServerTelemetryManager {
+    logger?.verbose("initializeServerTelemetryManager called");
+    const telemetryPayload: ServerTelemetryRequest = {
+        clientId: clientId,
+        correlationId: correlationId,
+        apiId: apiId,
+        forceRefresh: forceRefresh || false,
+        wrapperSKU: browserStorage.getWrapperMetadata()[0],
+        wrapperVer: browserStorage.getWrapperMetadata()[1],
+    };
 
-    /**
-     *
-     * @param apiId
-     * @param correlationId
-     * @param forceRefresh
-     */
-    protected initializeServerTelemetryManager(
-        apiId: number,
-        forceRefresh?: boolean
-    ): ServerTelemetryManager {
-        this.logger.verbose("initializeServerTelemetryManager called");
-        const telemetryPayload: ServerTelemetryRequest = {
-            clientId: this.config.auth.clientId,
-            correlationId: this.correlationId,
-            apiId: apiId,
-            forceRefresh: forceRefresh || false,
-            wrapperSKU: this.browserStorage.getWrapperMetadata()[0],
-            wrapperVer: this.browserStorage.getWrapperMetadata()[1],
-        };
-
-        return new ServerTelemetryManager(
-            telemetryPayload,
-            this.browserStorage
-        );
-    }
+    return new ServerTelemetryManager(telemetryPayload, browserStorage);
+}
 
-    /**
-     * Used to get a discovered version of the default authority.
-     * @param params {
-     *         requestAuthority?: string;
-     *         requestAzureCloudOptions?: AzureCloudOptions;
-     *         requestExtraQueryParameters?: StringDict;
-     *         account?: AccountInfo;
-     *        }
-     */
-    protected async getDiscoveredAuthority(params: {
+/**
+ * Used to get a discovered version of the default authority.
+ * @param params - Configuration object containing authority and cloud options
+ * @param params.requestAuthority - Optional specific authority URL to use
+ * @param params.requestAzureCloudOptions - Optional Azure cloud configuration options
+ * @param params.requestExtraQueryParameters - Optional additional query parameters
+ * @param params.account - Optional account info for instance-aware scenarios
+ * @param config - Browser configuration containing auth settings
+ * @param correlationId - Unique identifier for correlating requests
+ * @param performanceClient - Performance monitoring client instance
+ * @param browserStorage - Browser cache manager instance
+ * @param logger - Logger instance for tracking operations
+ * @returns Promise that resolves to a discovered Authority instance
+ */
+export async function getDiscoveredAuthority(
+    params: {
         requestAuthority?: string;
         requestAzureCloudOptions?: AzureCloudOptions;
         requestExtraQueryParameters?: StringDict;
```

Comment:
Out of scope for this PR but in light of recent conversations around EQP let's consider accepting instanceAware here instead of the whole object and have a separate helper we can use to get the correct value. Can we put something on the backlog?